troupon/static/scss/partials/_sections.scss

```diff
@@ -0,0 +1,374 @@
+
+/* ==========================================================================
+Sections
+========================================================================== */
+
+//entire body:
+
+html, body, .viewport-container {
+	height: 100%;
+}
+
+body {
+	position: relative;
+	min-width: 360px;
+
+	background-color: color(body-bg);
+}
+
+//subscribe bar:
+
+#subscribe-bar {
+	position: relative;
+	margin-top: 10px;
+
+	.tag-line {
+		margin: 0px 5px;
+		span {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3